### ISSUE TITLE
Code Release

### ISSUE THREAD

**Issue by eugenelawrence** [hlzhu09/Hierarchical-Human-Parsing#1]
When will you release the code?

**Comment by eugenelawrence** [comment]
Any Update? It seems the released code cannot reproduce the 73.12 miou on Pascal dataset.

**Comment by Adnan-annan** [comment]
@eugenelawrence are you able to run and train this model ?

**Comment by eugenelawrence** [comment]
> @eugenelawrence are you able to run and train this model ?

No, the authors do not provide the pre-trained model. Thus I cannot run this model.
And I have tried to re-implement the model based on this model, but I failed to reproduce the performance mentioned in the paper.

**Comment by wenguanwang** [comment]
@hlzhu09 Hi，Hailong, could you please release the trained weight ASAP? I told you so many times

**Comment by wenguanwang** [comment]
> Any Update? It seems the released code cannot reproduce the 73.12 miou on Pascal dataset.

During training, we use some tricks in semantic segmentation and spend a lot of time finetuning some hyper parameters. I plan to release the training code. But the student has graduated and I am quite not sure when will he can make it done …: (

**Comment by eugenelawrence** [comment]
> > Any Update? It seems the released code cannot reproduce the 73.12 miou on Pascal dataset.
> 
> During training, we use some tricks in semantic segmentation and spend a lot of time finetuning some hyper parameters. I plan to release the training code. But the student has graduated and I am quite not sure when he can make it done …: (

Thanks! Hope to see the training code ASAP.
